frontend/lib/src/components/widgets/TextInput/TextInput.tsx

```diff
@@ -226,6 +230,9 @@ class TextInput extends React.PureComponent<Props, State> {
               },
             },
             Root: {
+              props: {
+                "data-testid": "textInputRootElement",
```

Comment:
Related note: Since we are adding a lot of `data-testid` as part of the migration to RTL, maybe it is worth it to unify it a bit on how we name data-testid's 🤔 I have seen a lot of different naming styles within the codebase. The quasi-standard for the main component seems to be `st<ComponentName>`. But for subcomponents, it's very mixed.

Reply:
Good point, think we should definitely gather consensus 👍🏼